security/aporeto/docs/README.md

```diff
@@ -1,3 +1,6 @@
+---
+description: Learn how to create a Zero Trust environment and apply network security best practices.
+---
```

Comment:
just to prevent some extra work later on I'd recommend adding this front matter as well.

```md
---
description: Learn how to create a Zero Trust environment and apply network security best practices.
tags:
- aporeto
- security
- firewall
---
```

Reply:
Done.